text/0000-north-star.md

```diff
@@ -0,0 +1,466 @@
+  product, and establish a clear public narrative**.
+
+[s]: http://blog.rust-lang.org/2014/10/30/Stability.html
+
+All told, there's a lot of room to do better in establishing, communicating, and
+driving the vision for Rust.
+
+This RFC proposes changes to the way The Rust Project plans its work,
+communicates and monitors its progress, directs contributors to focus on the
+strategic priorities of the project, and finally, delivers the results of its
+effort to the world.
+
+The changes proposed here are intended to work with the particular strengths of
+our project - community development, collaboration, distributed teams, loose
+management structure, constant change and uncertanty. It should introduce
+minimal additional burden on Rust team members, who are already heavily
+overtasked. The proposal does not attempt to solve all problems of project
+management in Rust, nor to fit the Rust process into any particular project
+mamnagement structure. Let's make a few incremental improvements that will have
+the greatest impact, and that we can accomplish without disruptive changes to
+the way we work today.
+
+# Detailed design
+[design]: #detailed-design
+
+Rust's roadmap will be established in year-long cycles, where we identify up
+front the most critical problems facing the project, formulated as _problem
+statements_. Work toward solving those problems, _goals_, will be planned in
+quarter-long cycles by individual teams. For the purposes of reporting the
+project roadmap, goals will be assigned to _quartely milestones_, and where
+these goals result in stable features the Rust version in which they become
+stable will be estimated as well. Along the way, teams will be expected to
+maintain _tracking issues_ that communicate progress toward the project's goals.
+
+At the end of the year we will deliver a public facing retrospective, which is
+intended as a 'rallying point'. Its primary purposes are to create anticipation
+of a major event in the Rust world, to motivate (rally) contributors behind the
+goals we've established to get there, and generate a big PR-bomb where we can
+brag to the world about what we've done. It can be thought of as a 'state of the
+union'. This is where we tell Rust's story, describe the new best practices
+enabled by the new features we've delivered, celebrate those contributors who
+helped achieve our goals, honestly evaluate our performance, and look forward to
+the year to come.
+
+## Summary of terminology
+
+Key terminology used in this RFC:
+
+- _problem statement_ - A description of a major issue facing Rust, possibly
+  spanning multiple teams and disciplines. We decide these together, every year,
+  so that everybody understands the direction the project is taking. These are
+  used as the broad basis for decision making throughout the year, and are
+  captured in the yearly "north star RFC", and tagged `R-problem-statement`
+  on the issue tracker.
```

Comment:
Which repo are you proposing using the issue tracker on? Rust, RFCs, something new? (I would like something new - I feel the existing repos are already overloaded)